tests/h/panels/navbar_test.py

```diff
@@ -93,3 +98,10 @@ def user(self, factories):
     def req(self, pyramid_request):
         pyramid_request.user = None
         return pyramid_request
+
+    @pytest.fixture
+    def svc(self, pyramid_config, user):
```

Comment:
To avoid confusion if the navbar panel ends up depending on other services in future, I would suggest naming this `list_groups_svc`.